### PR TITLE
TCVP-2176 Added mapping for noticeOfHearingYn

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

TCVP-2176 - Added noticeOfHearingYn to the TCO OpenAPI spec 
- updated JJDisputeMapper
- updated JJDisputeMapperTest

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

mvn verify

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java [hunk 1]
```diff
@@ -54,6 +54,7 @@ public abstract class JJDisputeMapper extends BaseMapper {
 	@Mapping(source = "lawyerGiven3Nm", target = "lawyerGivenName3")
 	@Mapping(source = "lawyerSurnameNm", target = "lawyerSurname")
 	@Mapping(source = "noticeOfDisputeGuid", target = "noticeOfDisputeGuid")
+	@Mapping(source = "noticeOfHearingYn", target = "noticeOfHearingYn")
```
Comment (karahanb) [inline src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java:57]: As far as I know this is not needed as source and target have the same name. You may remove it if you like.

**Review by karahanb — APPROVED** [review]
Great work.

File: src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java (hunk 1 above)
Comment (KDNeufeld) [reply src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java:57]: Yes, I think you're right, its not needed if the names are the same. I just had it in there to be explicit so we have a complete enumerated list of fields that are mapped (there are several that are unmapped yet).

Comment (karahanb) [reply src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/JJDisputeMapper.java:57]: Oh okay, that makes sense.

**Pull request merged**